modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/util/VersionUtil.java

```diff
@@ -78,7 +78,7 @@ else if (label.startsWith("sp")) {
 	}
 
 	private static final Pattern _dxpVersionPattern = Pattern.compile(
-		"^[0-9]\\.[0-9]\\.[1-9][0-9](\\.((e|f)p)?[0-9]+(-[0-9]+)?)?$");
+		"^[0-9]\\.[0-9]\\.[1-9][0-9](-[0-9])?(\\.((e|f)p)?[0-9]+(-[0-9]+)?)?$");
```

Comment:
Hey @Seiphon  @simonjhy  we need to also support versions like `7.3.10-test` as well.  Also there is a VersionUtil test unit test that should be updated as well.  Thanks.